frontend/src/components/Register/Register.css

```diff
@@ -0,0 +1,102 @@
+.register {
```

Comment:
Seems like this holds a ton of similarity to Login.css? I wouldn't bother trying to merge them at this point, we probably won't be messing around much with the login anyway

Reply:
It's pretty much the same thing honestly, I can merge them tomorrow as the code/css is mostly the same (I copy pasted it)